pages/join.tsx

```diff
@@ -1,9 +0,0 @@
-import React, { useEffect } from 'react';
-
-export default function Join(): null {
```

Comment:
Wanted to double check if this is being used anywhere else? The preview looked good to me on a quick smoke test.

Reply:
I couldn't say. Just trying to get the analytics in.